apps/dokuha-roadmap-frontend/src/app/details/[contentId]/page.tsx

```diff
@@ -0,0 +1,182 @@
+'use client';
+
+import { useParams, useRouter } from 'next/navigation';
+import { useEffect, useState } from 'react';
+
+import Button from "@/components/ui/Button";
+
+export type postData = {
+  currentPage: number;
+  note: string;
+}
```

Comment:
スコープは狭くできるならそのほうが良さそう！
```suggestion
type postData = {
  currentPage: number;
  note: string;
}
```